client/src/i18n/fr/report.json

```diff
@@ -242,7 +242,7 @@
     },
     "STOCK":{
       "TITLE" : "Rapport de stock",
-      "DESCRIPTION" : "Ce rapport affiche les stocks de produits ansi que leur moyenne de consommation et état",
+      "DESCRIPTION" : "Ce rapport montre le stock de produits, leur consommation mensuelle moyenne et indique si une commande supplémentaire est nécessaire à la date spécifiée.  Les articles sont classés par groupe de stock, puis par leur nom.  Par défaut, tous les produits de chaque dépôt sont affichés, mais il est possible de les filtrer en spécifiant un produit ou un dépôt individuel.",
```

Comment:
Tried with this translation **Ce rapport indique le stock des produits, leur consommation mensuelle moyenne et indique si une commande est nécessaire à la date spécifiée. Les articles sont classés par groupe d'inventaire, puis par leur nom. Par défaut, tous les produits de chaque dépôt sont affichés, mais ceux-ci peuvent être filtrés en spécifiant un produit ou un dépôt individuel.**